app/javascript/Greeting.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from 'react';
```

Comment:
- [ ] [OPTIONAL] It would be better to create a component folder and move this component into the newly created folder